src/pages/winterGardening.js

```diff
@@ -0,0 +1,30 @@
+import  ReactPlayer from "react-player";
+import {videoData} from "../data/videodataList";
+import "../displaycomponents/category/category.css";
+import LikeButton from "../displaycomponents/likeButton/likefunctionality.js";
+import WatchLaterButton from "../displaycomponents/watchLaterButton/watchLater";
+function WinterVideoList(){
+
+    return(<div>
+        <h3>Winter Special Gardening Videos</h3>
+        <div className="video-list-display">
+        {
+                videoData.map((item)=>{
+                    if(item.category==="winter"){
+                        return (<div className="video-style">
+                            <ReactPlayer url= {item.url} controls={true} width="420px" height="250px" />
+                            <div className="video-btns">
+                            <LikeButton item={item}/>
+                            <WatchLaterButton item={item}/>
+                            </div>
+                           </div>) 
+                    }
+                  return (<div></div>);
+                }) 
```

Comment:
You can create generalized component and you can pass props so you don't need to create different components like this